receiver/postgresqlreceiver/client.go

```diff
@@ -102,41 +101,32 @@ func sslConnectionString(tls configtls.TLSClientSetting) string {
 	return conn
 }
 
-func newPostgreSQLClient(conf postgreSQLConfig) (*postgreSQLClient, error) {
+func (c postgreSQLConfig) DSN() (string, error) {
```

Comment:
DSN? Can we use a more expressive name?

Reply:
Sure, it's a bit cryptic indeed. Maybe `ConnectionString`?

Reply:
Sure, sounds good to me.

Reply:
Done